README.md

```diff
@@ -138,9 +139,23 @@ dep version //to verify that it is installed correctly
 
 git clone https://github.com/cloud-ark/kubeprovenance.git $GOPATH/src/github.com/cloud-ark<br/>
 cd $GOPATH/src/github.com/cloud-ark/kubeprovenance <br/>
+
+go get github.com/emicklei/go-restful<br/>
```

Comment:
Curious as to why do we need to separately do 'go get' for these libraries?

Can we not include them in Gopgk.toml and have them installed by dep ensure?

Reply:
Sure !